app/src/main/java/org/p2p/wallet/sdk/facade/RelaySdkFacade.kt

```diff
@@ -1,16 +1,19 @@
 package org.p2p.wallet.sdk.facade
 
+import com.google.gson.annotations.SerializedName
 import org.p2p.solanaj.model.types.RecentBlockhash
 import org.p2p.solanaj.utils.crypto.Base64String
 import org.p2p.solanaj.utils.crypto.encodeToBase58
 import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
 import org.p2p.wallet.sdk.RelaySdk
 import org.p2p.wallet.sdk.facade.mapper.SdkMethodResultMapper
 import org.p2p.wallet.utils.Base58String
-import org.p2p.wallet.utils.toBase58Instance
 import kotlinx.coroutines.withContext
 
-class SdkSignedTransaction(val transaction: Base58String)
+class SdkSignedTransaction(
```

Comment:
if it's network model, please add Response to class name

Reply:
fixed! thanks